dd-java-agent/instrumentation/jdbc/src/main/java/datadog/trace/instrumentation/jdbc/StatementInstrumentation.java

```diff
@@ -44,6 +44,14 @@ public StatementInstrumentation() {
   public String[] helperClassNames() {
     final List<String> helpers = new ArrayList<>(JDBCConnectionUrlParser.values().length + 9);
 
+    helpers.add(packageName + ".normalizer.Token");
```

Comment:
This code is duplicated 3 times. Can it be extracted and reused?

Reply:
None of the other decorators do this, so out of muzzle superstition I follow their pattern.